init.rb

```diff
@@ -50,7 +50,7 @@
 
 #This check is needed because the overrides file will override the FLAVOR selection.
 #Furthermore a selection != stable can cause a inconsistent layout (cause by in_flavor system in the package_sets)
-if File.exists?(File.join(Autoproj.root_dir, "autoproj", "overrides.d", "25-release.yml")) && current_flavor.branch != "stable" 
+if ENV['ROCK_RC'] != '1' && File.exists?(File.join(Autoproj.root_dir, "autoproj", "overrides.d", "25-release.yml")) && current_flavor.branch != "stable" 
```

Comment:
I would suggest to increase verbosity, i.e. something along the following lines:
```
if ENV['ROCK_RC'] == 1
    Autoproj.warn "You are operating on a Rock Release Candidate"
elsif ...
end